scripts/generate-style-code.js

```diff
@@ -6,12 +6,14 @@ const spec = require('mapbox-gl-style-spec').latest;
 var parseCSSColor = require('csscolorparser').parseCSSColor;
 
 global.camelize = function (str) {
+  str = str === undefined ? 'undefined' : str;
```

Comment:
Yeah, I see the `Alignment::Undefined`. Why does a JS `undefined` value pass through `camelize` though?